Signpost to adjustments and disabilities pages (#4739)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/school-placements.md

Your school leadership team and colleagues will be there to support you. Your placement schools may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.

### Adjustments and accessibility

> Providers and schools can make adjustments during your placements if you need support during your placement. This includes support for neurodiversity, long-term physical or mental health conditions, or other accessibility needs. Find out more about [adjustments to help you train](/train-to-be-a-teacher/accessibility-adjustments).

Reviewer: ```suggestion
Providers and schools can make adjustments if you need support during your placements. This includes support if you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs. Find out more about [adjustments to help you train](/train-to-be-a-teacher/accessibility-adjustments).
```
